tests/provision.fmf

```diff
@@ -0,0 +1,3 @@
+standard-inventory-qcow2:
+  qemu:
+    m: 2048
```

Comment:
This is the default https://pagure.io/standard-test-roles/blob/master/f/inventory/standard-inventory-qcow2#_336

Reply:
It seems to me this machine, where OOM killer got runned, had only 1GB of RAM:
https://fedorapeople.org/groups/linuxsystemroles/logs/linux-system-roles-ha_cluster-pull-linux-system-roles_ha_cluster-4-18066ad-centos-8-20210216-121111/artifacts/centos-8.qcow2.guest.log
That's why I set it to 2GB.

Reply:
> That's why I set it to 2GB.

Hmm - that's disturbing that we cannot rely on the default set in standard-inventory-qcow2!  Sounds like a bug in that script.

So ok - for ha_cluster - let's keep this provision.fmf just to make sure

Reply:
@richm see my last comment